src/pages/about.js

```diff
@@ -0,0 +1,229 @@
+import React from "react";
+import { graphql } from "gatsby";
+import { makeStyles } from "@material-ui/core/styles";
+import Img from "gatsby-image";
+import Layout from "../components/layout";
+import SEO from "../components/seo";
+import Typography from '@material-ui/core/Typography';
+
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardActions from '@material-ui/core/CardActions';
+
+import GitHubIcon from '@material-ui/icons/GitHub';
+import Button from '@material-ui/core/Button';
+
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemAvatar from '@material-ui/core/ListItemAvatar';
+import Avatar from '@material-ui/core/Avatar';
+
+
+const useStyles = makeStyles({
+  aboutPage: {
+    display: 'grid',
+    placeItems: 'center',
+    gridTemplateRows: '1fr auto',
+    gridTemplateColumns: '100%'
+  },
+  aboutCards: {
+    margin: '16px',
+    '& .MuiCard-root:not(:last-child)': {
+      marginBottom: '16px'
+    }
+  },
+  enrtf: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    alignItems: 'center',
+    margin: '0px -8px -8px',
+    '& > *': {
+      margin: '8px'
+    },
+  },
+  media: {
+    marginTop: 'auto',
+    marginBottom: 'auto'
+  },
+  enrtfText: {
+    minWidth: '60%',
+    flex: 1
+  }
+});
+
+
+const About = ({ data: { enrtfLogo, dynamicUsers, staticUsers } }) => {
+  const classes = useStyles();
+  const staticUserList = staticUsers.data;
+  return (
+    <Layout>
+      <SEO title="About" />
+      <div className={classes.aboutPage}>
+        <div className={classes.aboutCards}>
+
+          <Card>
+            <CardContent>
+              <Typography gutterBottom variant="h5" component="h2">Pollinator Facts</Typography>
```

Comment:
I just added the green plant icon we already had in pollinator-fact repository to the about page. I can switch it to the icon on buzz-about if we want to be consistent. In that case, it would be nice to have a green version of that bee icon to match the theme of pollinator-facts.